<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPersonnel</class>
 <widget class="QDialog" name="AddPersonnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Sprzedawca_tab">
    <attribute name="title">
     <string>Sprzedawca</string>
    </attribute>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Akceptuj">
        <property name="text">
         <string>Akceptuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Anuluj_sprzedawca">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QTextEdit" name="Nazwisko_sprzedawca_pisz">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Nazwisko_sprzedawca">
        <property name="text">
         <string>Nazwisko:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Wynagrodzenie_sprzedawca">
        <property name="text">
         <string>Wartość wynagrodzenia: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Imie_sprzedawca">
        <property name="text">
         <string>Imie: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="Wynagrodzenie_sprzedawca_pisz">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="Imie_sprzedawca_pisz">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::WidgetWidth</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="Doswiadczenie_sprzedawca_pisz">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Doswiadczenie_sprzedawca">
        <property name="text">
         <string>Doświadczenie w latach: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Mechanik_tab">
    <attribute name="title">
     <string>Mechanik</string>
    </attribute>
    <widget class="QGroupBox" name="horizontalGroupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Akceptuj_mechanik">
        <property name="text">
         <string>Akceptuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Anuluj_mechanik">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="formGroupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Specjalizacja_mechanik">
        <property name="font">
         <font>
          <family>Avenir Next</family>
         </font>
        </property>
        <property name="text">
         <string>Specjalizacja: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="Nazwisko_mechanik_pisz">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Nazwisko_mechanik">
        <property name="text">
         <string>Nazwisko: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Wynagrodzenie_mechanik">
        <property name="text">
         <string>Wartość wynagrodzenia: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Spacjalizacja_mechanik_wybierz"/>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="Imie_mechanik_pisz">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Imie_mechanik">
        <property name="text">
         <string>Imie:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="Wynagrodzenie_mechanik_pisz">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
